### PR TITLE
Give the possibility to pass extensions options to the config template

### PR DESCRIPTION
**Pull request by nremond**

### CODE REVIEW

**Comment by otakup0pe** [comment]
Looks like test failing due to missing defaults ?

**Comment by nremond** [comment]
@otakup0pe yes, sorry. Just fixed that.

File: defaults/main.yml [hunk 1]
```diff
@@ -683,3 +683,6 @@ postgresql_pgdg_dists:
 postgresql_version_terse: "{{ postgresql_version | replace('.', '') }}"
 postgresql_yum_repository_base_url: "http://yum.postgresql.org"
 postgresql_yum_repository_url: "{{ postgresql_yum_repository_base_url }}/{{ postgresql_version }}/{{ ansible_os_family | lower }}/rhel-{{ ansible_distribution_major_version }}-{{ ansible_architecture }}/pgdg-{{ postgresql_pgdg_dists[ansible_distribution] }}{{ postgresql_version_terse }}-{{ postgresql_version }}-2.noarch.rpm"
+
+# List of options from loaded extensions (optional)
+postgresql_extensions_options: []
```
Comment (otakup0pe) [inline defaults/main.yml]: What is an example of this format? It looks like a free-form way of injecting config entries?

Comment (nremond) [inline defaults/main.yml]: Right. My use-case is the following, I am using the `pg_stat_statements` extension and I would like to be able to add `pg_stat_statements.track = all` in the config. 

Does this sound good to you or would you prefer passing something like `{ key: 'pg_stat_statements.track', value: 'all' }` ?

**Comment by nremond** [comment]
I have updated my PR with a dictionary and an example of how to use it in the README.
What do you think of it @otakup0pe

**Pull request closed without merging**